docs/DeveloperGuide.md

```diff
@@ -114,14 +114,14 @@ How the parsing works:
 * All `XYZCommandParser` classes (e.g., `AddCommandParser`, `DeleteCommandParser`, ...) inherit from the `Parser` interface so that they can be treated similarly where possible e.g, during testing.
 
 ### Model component
-**API** : [`Model.java`](https://github.com/se-edu/addressbook-level3/tree/master/src/main/java/seedu/address/model/Model.java)
+**API** : [`Model.java`](https://github.com/se-edu/addressbook-level3/tree/master/src/main/java/seedu/telegramHandle/model/Model.java)
 
 <img src="images/ModelClassDiagram.png" width="450" />
 
 
 The `Model` component,
 
-* stores the address book data i.e., all `Person` objects (which are contained in a `UniquePersonList` object).
+* stores the telegramHandle book data i.e., all `Person` objects (which are contained in a `UniquePersonList` object).
```

Comment:
This seems inappropriate, about calling the address book a telegramHandle book.

(This is repeated in parts below as well, so be aware if a fix is to be done.)

Reply:
Thank you. will change